solidity/build/contracts/Oracle.json

```diff
@@ -355,84 +355,84 @@
       "type": "function"
```

Comment:
Ooops, we should .gitignore and rm this.

Reply:
This was added for https://github.com/smartcontractkit/chainlink/pull/950.